scripts/quote-board.coffee

```diff
@@ -0,0 +1,42 @@
+# Description:
+#  Quote Board Interface
+#
+# Dependencies:
+#  None
+#
+# Commands:
+#  !quote <name> said <quote> - Adds that quote attributed to that person to the DB
+#  !quote_me_as <name> - Adds <name> to be your full name for quotes attributed to you
+#
+# Notes:
+#  None
+#
+# Author:
+#  @murph33
+
+QUOTE_URL = "https://quote-board.herokuapp.com/api/v1/quotes"
+PERSON_URL = "https://quote-board.herokuapp.com/api/v1/people"
+
+module.exports = (robot) ->
+  robot.hear /![q|Q]uote_me_as (.*)/i, (msg) ->
+    data =
+      person:
+        slack_name: "@#{msg.message.user.name}"
+        full_name: msg.match[1]
+    data = JSON.stringify(data)
+    msg.http(PERSON_URL)
+      .header('Content-Type', 'application/json')
+      .post(data) (err, res, body) ->
+        msg.send('It worked!') if res.statusCode is 201
+        msg.send('It did not work') if res.statusCode is 400
+  robot.hear /![q|Q]uote (.*) said (.*)/i, (msg) ->
```

Comment:
If your quote contains the word "said" this regexp will produce surprising results.

https://blog.mariusschulz.com/2014/06/03/why-using-in-regular-expressions-is-almost-never-what-you-actually-want